backend/curriculum_tracking/models.py

```diff
@@ -1477,9 +1477,9 @@ class BurndownSnapshot(models.Model):
     def create_snapshot(Cls, user):
         match = Cls.objects.filter(
             user=user,
-            timestamp__gte=timezone.now()
+            timestamp__gte=timezone.now()  
             - timedelta(hours=Cls.MIN_HOURS_BETWEEN_SNAPSHOTS),
-        ).first()
+        ).last()
```

Comment:
So the tests pass on my side whether this is set to last() or first() so I'm not sure what this change is doing?
Or if maybe the tests aren't capturing what a board regen would look like? To be honest I'm not sure how to test a board regen so I can't actually say if this isn't the right way, lol